### PR TITLE
master-lps-126859

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-126873
https://issues.liferay.com/browse/LPS-126862
https://issues.liferay.com/browse/LPS-126874
https://issues.liferay.com/browse/LPS-126863
https://issues.liferay.com/browse/LPS-126868
https://issues.liferay.com/browse/LPS-126875
https://issues.liferay.com/browse/LPS-126878
https://issues.liferay.com/browse/LPS-126872
https://issues.liferay.com/browse/LPS-126871
https://issues.liferay.com/browse/LPS-126869
https://issues.liferay.com/browse/LPS-126861
https://issues.liferay.com/browse/LPS-126876
https://issues.liferay.com/browse/LPS-126877
https://issues.liferay.com/browse/LPS-126866

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
This SF issue is related to the stub, not my code issue.

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/documentsadministration/fields/DMImageField.testcase [hunk 1]
```diff
@@ -96,61 +337,215 @@ definition {
 	test SetImageFieldAsRepeatable {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98073 SetImageFieldAsRepeatable pending implementation
+		// TODO LPS-98073 SetImageFieldAsRepeatable pending implementation (blocker by LPS-127049)
```
Comment (austinchiang) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/documentsadministration/fields/DMImageField.testcase:340]: @Jeremy-Ch thanks for adding this!

**Comment by austinchiang** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/1441

**Pull request closed without merging**